controls/cis_ocp_1_4_0/section-5.yml

```diff
@@ -103,13 +103,16 @@ controls:
     controls:
     - id: 5.3.1
       title: Ensure that the CNI in use supports Network Policies
-      status: pending
-      rules: []
+      status: partial
```

Comment:
why partial?

Reply:
I looked at the rule and updated it to `automated`.